### DISCUSSION TITLE
Some questions about Moltres app

### DISCUSSION THREAD

**Discussion by SomnusYu** [idaholab/moose#26271] [Q&A General]
Hi all,

I would like to know if there are some theory manuals about the Moltres app. I found some introduction (https://arfc.github.io/software/moltres/wiki/kernels/) but I can not read it, which is shown:
![image](https://github.com/idaholab/moose/assets/118275988/6f2dc65f-c5b8-4bc0-9ad6-907bb75cbbfd)

By the way, I have another question. If it is possible to solve some neutronic problems in 3D geometry using the Moltres app?

Thanks!

**Comment by GiudGiud** [comment]
Hello

You are on the right website
From this link http://arfc.github.io/software/moltres/
you can go on their wifi and the equations are formatted better there.

I would think you can do 3D yes. it will be more expensive computationally. 
You could also ask for access to the NEAMS neutronics code: Griffin, here: https://ncrcaims.inl.gov/Identity/Account/Login
Do you have collaborations with INL or ANL / any US institutions? You'll need a sponsor to request access

Guillaume

**Reply by SomnusYu** [reply]
Thank you very much! I do not have any collaborations with US institutions. In this case, is the Moltres my best choice to solve 3D neutronic problems?

**Reply by GiudGiud** [reply]
Depends on the scale of your project tbh. I think it's good to get started, and when you hit its limitations you can decide if you want to add the features you need to Moltres or using Cardinal (https://github.com/neams-th-coe/cardinal) or work on coupling in a non-MOOSE neutronics codes

**Reply by lindsayad** [reply]
And because it's open source you can always contribute any updates you make back to it so that it continues to grow

**Reply by SomnusYu** [reply]
Thanks! Of course, I'm glad to make Moltres better.
